src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/QueryCQLFilters/QueryCQLFilters.xaml.cs

```diff
@@ -196,5 +177,13 @@ private async void ApplyQuery_Click(object sender, RoutedEventArgs e)
                 LoadingProgressBar.Visibility = Visibility.Hidden;
             }
         }
+
+        private void DateBox_Checked(object sender, RoutedEventArgs e)
```

Comment:
This could be made a bit cleaner if done through binding in the XAML. Just a suggestion